app/models/preservica/content_object.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class ContentObject
+  include PreservicaObject
+
+  def self.where(options)
+    preservica_client = options[:preservica_client] || PreservicaClient.new(options[:admin_set_key])
+    ContentObjecct.new(preservica_client, options[:id])
+  end
+
+  def initialize(preservica_client, id)
+    @preservica_client = preservica_client
+    @id = id
+  end
+
+  def generations
```

Comment:
Is this loading only the active generation?  If so, please rename to `active_generation`

Reply:
changes name to active_generations